JsonLogic/Rule.cs

```diff
@@ -72,7 +72,7 @@ public abstract class Rule
 /// <summary>
 /// Provides serialization for all <see cref="Rule"/> derivatives.
 /// </summary>
-public class LogicComponentConverter : JsonConverter<Rule>
+public class LogicComponentConverter : JsonConverter<Rule>, Json.More.IJsonConverterReadWrite<Rule>
```

Comment:
Why are we adding this to all of the converters?  I thought this was just to support the `object? Read()` used in JsonSchema.

This interface never gets used on these converters.

Reply:
I'll revert these changes.